### DISCUSSION TITLE
installing onion on MM+ (extra folder) - question

### DISCUSSION THREAD

**Discussion by Greenboi27** [OnionUI/Onion#1045] [Q&A] [answered]
Hi, im green retro enjoyer, im trying to install onionos on my MM+.
Onion V4.2.0-RC  has 7 folders instead of 6, there is an extra "miyoo354"  folder that isnt mentioned in the instuction, should i paste it or ignore?

Thanks

**Comment by Aemiii91** [comment, marked as answer]
It's not important. It is used if you forget to transfer the `.tmp_update` folder.
